app/views/spree/layouts/_admin_body.html.haml

```diff
@@ -18,30 +18,43 @@
       %nav.columns.eleven{"data-hook" => "admin_login_navigation_bar"}
         = render :partial => 'spree/layouts/admin/login_nav'
 
-  %nav#admin-menu{"data-hook" => ""}
+  %nav.admin__menu{ data: { hook: '' }}
```

Comment:
BEM as already been discussed here: https://community.openfoodnetwork.org/t/some-ideas-for-improving-ofn-css/785/3
Maybe we should just go for it :+1: